include/picongpu/param/laser.param

```diff
@@ -21,15 +21,95 @@
 /** @file
  *
  * Configure laser profiles.
+ *
+ * Available profiles:
+ *  - None                : no laser init
+ *  - GaussianBeam        : Gaussian beam (focusing)
+ *  - PulseFrontTilt      : Gaussian beam with a tilted pulse envelope
+ *                          in 'x' direction
```

Comment:
Better describe which axes enclose the pulse-front tilt angle. At the moment the description is ambiguous.